mediorum/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+
+services:
+  db:
+    image: postgres
+    container_name: pg
+    ports:
+      - 5444:5432
+    volumes:
+      - ./.initdb:/docker-entrypoint-initdb.d
+    environment:
+      POSTGRES_PASSWORD: example
```

Comment:
this isolated flow is faster/lighter weight, but if we merge this I don't think it'll be possible to test the full stack locally with just `audius-compose up --storagev2`. I think it would be best if we just support one flow through audius-compose (even just a flag to only bring up mediorum containers + db), but I'm happy as long as audius-compose up still works. maybe all that's needed to accomplish that is to make the `db` container in audius-compose expose the port so mediorum can access it through localhost

Reply:
@theoilie I updated the compose file so that mediorum runs in compose using the postgres container.

I tested thusly:

```
docker compose up db -d
```

**If you have existing postgres state you'll have to create dbs manually**  (init scripts only run one time):

```
docker compose exec db psql -U postgres

CREATE DATABASE mediorum1;
CREATE DATABASE mediorum2;
CREATE DATABASE mediorum3;
CREATE DATABASE mediorum4;
CREATE DATABASE mediorum5;
```

Finally:

```
docker compose up storagev2-1 --build
```


After some head scratching on docker networking, hosts, config, peer list, etc I decided the easiest thing to do was run in a single container using the `devCluster` entry point... similar to how tests work.